articles/using-picasso-in-android/index.md

```diff
@@ -0,0 +1,382 @@
+This article will illustrate the different features offered by Picasso in Android applications.
+
+### Introduction
+Visual representation in mobile application development is a growing UI/UX trend. In this current era, most of our minds seem to understand either graphics in motion or pictures rather than words. With time, most mobile applications, such as Instagram, Facebook, Twitter, LinkedIn, etc., started implementing visual representation. It was not perfect the first time. Images were of low quality, could neither be downloaded nor edited, and screens would "freeze" as pictures loaded. As much as it was an improvement in mobile development, it was still hectic. Developers had to solve these problems. Many visual representation libraries such as `Glide`, `Picasso`, `Image loader`, `Fresco`, etc ,were then developed. In his article, we will discuss **Picasso**. **Picasso** is a powerful image loading, downloading, and caching library developed by Square. Picasso offers better image quality, disk caching, and simpler syntax to carry out various library features.
+
+### Uses of Picasso
+Picasso has the following features:
+- Image loading.
+- Resizing and Scaling of images. 
+- Center Cropping images.
+- Rotation and Transformation of images. 
+- Placeholder and Error images
+- Priority requests
```

Comment:
You could explain these uses more to give the user a richer understanding of the uses.